conduit/src/Data/Conduit/Combinators.hs

```diff
@@ -7,21 +7,18 @@
 {-# LANGUAGE NoImplicitPrelude         #-}
 {-# LANGUAGE NoMonomorphismRestriction #-}
 {-# LANGUAGE BangPatterns #-}
--- | This module is meant as a replacement for Data.Conduit.List.
```

Comment:
It's worth adding a mention to the `Conduit` module. Note that it imports the unqualified names (with the `C` suffix).